app/views/patients/show.html.erb

```diff
@@ -249,22 +276,36 @@
         </tr>
       </thead>
       <tbody>
-        <% @patient.close_contacts.order('created_at').each do |cc| -%>
-        <tr>
-          <td><%= cc.first_name %></td>
-          <td><%= cc.last_name %></td>
-          <td><%= cc.primary_telephone.present? && cc.primary_telephone.length > 10 ? cc.primary_telephone&.gsub('+1', '')&.gsub(/\D/, '')&.insert(3, '-')&.insert(7, '-') : '' %></td>
-          <td><%= cc.email %></td>
-          <td><%= cc.contact_attempts %></td>
-          <td>
-            <div class="btn-group btn-group-sm" role="group">
-              <%= react_component('close_contact/CloseContact', { authenticity_token: form_authenticity_token, patient: @patient, close_contact: cc, can_enroll_patient_close_contacts: current_user.can_enroll_patient_close_contacts? }) %>
-            </div>
-          </td>
-        </tr>
+        <% if @close_contacts.length > 0 %>
+          <% @close_contacts.each do |cc| -%>
+          <tr>
+            <td><%= cc.first_name %></td>
+            <td><%= cc.last_name %></td>
+            <td><%= cc.primary_telephone %></td>
```

Comment:
Why was the logic around the primary telephone removed?

Reply:
think this was probably a rebase miss - fixed now